### DISCUSSION TITLE
CodeBooga + text-generation-webui what does he have in store?

### DISCUSSION THREAD

**Discussion by nbollman** [oobabooga/text-generation-webui#4335] [Q&A]
https://huggingface.co/oobabooga/CodeBooga-34B-v0.1
Im excited.

**Comment by FoobarProtocol** [comment]
I am too. Saw this name mentioned on github in passing convo. Decided to look it up on the off chance it existed and then stumbled on your post here on github. 

Thank you my friend. You are a gentleman and a scholar and you've done a good deed whether you know it or not. Same goes for the teams that contributed to the building of that new model.
